app/Donations.tsx

```diff
@@ -409,9 +483,65 @@ export default function Organizations() {
                 <h3 className="sr-only">Categories</h3>
 
                 {filters.map((section) => createFilterElement(section))}
-                {sortedOrganizations.length !== organizations.length && (
-                  <p className="text-gray-400 text-center mt-4">
-                    Displaying {sortedOrganizations.length} of{" "}
+                {filters !== initialFilters ? (
```

Comment:
Everything is flawless, thank you, @nhestrompia!

Only thing is that, the intention here is to hide reset and copy buttons when there is no filters are set. However, although the user resets the filters, this check returns true and still displays the buttons on the page.

If this behavior is fixed, we are good to go.

Reply:
Fixed this problem. Let me know if more adjustments needed

Reply:
Thank you @nhestrompia!

LGTM, merging now.